content/en/opentelemetry/config/environment_variable_support.md

```diff
@@ -97,6 +108,46 @@ Disable the SDK for all signals<br>
   - `false`|`true`<br>
 **Ruby & Go SDKs**: The OpenTelemetry SDK activates automatically upon import and configuration, so this setting is not applicable.
 
+## OTLP Exporter configuration
+Datadog SDKs support the following [OpenTelemetry OTLP Exporter][13] options.
+
+`OTEL_EXPORTER_OTLP_PROTOCOL`
+: **Description**: Specifies the transport protocol to use. Signal-specific variables (like `OTEL_EXPORTER_OTLP_METRICS_PROTOCOL`) are also supported. <br>
+**Accepted values**: `grpc`, `http/protobuf`, `http/json`. <br>
+**Default**: SDK-dependent, either `http/protobuf` or `grpc`.
+
+`OTEL_EXPORTER_OTLP_ENDPOINT`
+: **Description**: Specifies the URL for sending OTLP data. <br>
+**Default (gRPC)**: `http://localhost:4317`. <br>
+**Default (HTTP)**: `http://localhost:4318/v1/metrics`.
+
+`OTEL_EXPORTER_OTLP_HEADERS`
+: **Description**: Specifies a comma-separated list of key-value pairs to be used as headers on all outgoing OTLP requests (for example, `api-key=key,other-config=value`). Signal-specific headers (like `OTEL_EXPORTER_OTLP_METRICS_HEADERS`) take precedence.
+
+`OTEL_EXPORTER_OTLP_TIMEOUT`
+: **Description**: Specifies the timeout (in milliseconds) for a single outgoing OTLP request. <br>
+**Default**: `10000` (10s).
+
+## OpenTelemetry Metrics SDK configuration
+
+Datadog SDKs support the following OpenTelemetry Metrics SDK options.
+
+`OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE`
+: **Description**: Specifies the aggregation `temporality` to use for each instrument kind. <br>
```

Comment:
We can also link to this page for more details: https://opentelemetry.io/docs/specs/otel/metrics/sdk_exporters/otlp/#additional-environment-variable-configuration